libs/vkd3d/va_map.c

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright 2020 Philip Rebohle for Valve Software
```

Comment:
2021

Reply:
It's technically code from 2020 but can change I guess.